# oteodoro changes:
# Patch status:  testing
# For use for the distro's hardened profile
diff -urp clang.orig/lib/Frontend/InitPreprocessor.cpp clang/lib/Frontend/InitPreprocessor.cpp
--- clang.orig/lib/Frontend/InitPreprocessor.cpp	2021-09-28 19:40:49.027892121 -0700
+++ clang/lib/Frontend/InitPreprocessor.cpp	2021-09-28 19:40:15.829892652 -0700
@@ -1190,6 +1190,8 @@ static void InitializePredefinedMacros(c
     Builder.defineMacro("__GLIBCXX_BITSIZE_INT_N_0", "128");
   }
 
+  Builder.defineMacro("_FORTIFY_SOURCE", "2");			// Added by oteodoro to match hardened profile feature parity
+								// Added by oteodoro
   // Get other target #defines.
   TI.getTargetDefines(LangOpts, Builder);
 }
